kasa/discover.py

```diff
@@ -423,7 +423,7 @@ def _get_device_instance_legacy(data: bytes, config: DeviceConfig) -> SmartDevic
     def _get_device_instance(
         data: bytes,
         config: DeviceConfig,
-    ) -> SmartDevice:
+    ) -> Device:
```

Comment:
These will be always of type `smart.Device`, right?

Reply:
This could be `iot.Device` for the klap HS100 etc.